### ISSUE TITLE
ColorPicker: *.png missing

### ISSUE THREAD

**Issue by 102-97-98-105** [primefaces/primeng#13820]
### Describe the bug

By following the installation guide the png-files are missing, and the color picker is broken. (fore example the hue.png is missing)

### Environment

Angular CLI: 16.2.4
Node: 18.17.0
Package Manager: npm 9.8.1
Windows: 10
IDE: Intellij Community

### Angular version

16.2.4

### PrimeNG version

16.4.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.17.0

### Steps to reproduce the behavior

0. Angular application setup with sass for styling
1. Follow the installation guide.
2. Use the ColorPicker in a project.

### Expected behavior

![grafik](https://github.com/primefaces/primeng/assets/98735329/44dabf26-cd9b-45b5-ad12-9e88371a9e45)
Displaying that way, no error messages or anything else in dev tools.
Files are in the node_modules folder

**Comment by 102-97-98-105** [comment]
I found a fix: Add the following to the top level styles.sass

`/* You can add global styles to this file, and also import other style files */
.p-colorpicker-panel .p-colorpicker-color
    background: transparent url("./images/color.png") no-repeat left top


.p-colorpicker-panel .p-colorpicker-hue
   background: transparent url("./images/hue.png") no-repeat left top
`

The problem seems to be the styles generator that throws away the css styles of colorpicker-images.css

**Comment by SoyDiego** [comment]
I cannot reproduce.
I added in my project and all is working perfectly.

![image](https://github.com/primefaces/primeng/assets/19764334/add2cebb-7f27-46d8-b846-75f01bb97a83)

Maybe is better if you share a stackblitz or give more information.

Thanks

**Comment by sitterrus** [comment]
@SoyDiego i think the op used sass theme [https://github.com/primefaces/primeng-sass-theme/blob/main/theme-base/components/input/_colorpicker.scss](url) which doesn't have the backround style and indeed there is no background 
![image](https://github.com/primefaces/primeng/assets/40627292/f68ed235-24d3-48b9-9a9f-20c3eb60b072)


Shouldn't colorpicker-images.css be included in colorpicker component?[https://github.com/primefaces/primeng/blob/7216049475cdde34593d321181dbbb174e164dd8/src/app/components/colorpicker/colorpicker.ts#L74C38-L74C38](url)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz reproducer so we can identify the issue clearly?

**Comment by mrafiqk** [comment]
@cetincakiroglu 
Looks like the resource images are corrupted in the latest build. I replaced the images and added some style in css. It worked.

FYI:
style.css / style.scss
```css
/* You can add global styles to this file, and also import other style files */
.p-colorpicker-panel .p-colorpicker-color {
	background: transparent url("assets/img/custom-color.png") no-repeat left top
}


.p-colorpicker-panel .p-colorpicker-hue {
	background: transparent url("assets/img/custom-hue.png") no-repeat left top !important;
}
```

custom-color.png
![custom-color](https://github.com/user-attachments/assets/aefa5064-8e18-4bdc-a559-f058a8695583)

custom-hue.png
![custom-hue](https://github.com/user-attachments/assets/61460c5c-b283-49a4-9ebf-7a78246878d8)

**Comment by StefaniOSApps** [comment]
same

**Comment by faelpinho** [comment]
Same here. Both versions 17.18.10 and 17.18.11 (with Angular 18).

**Comment by Eagle-Tim** [comment]
> Hi,
> 
> Could you please share a stackblitz reproducer so we can identify the issue clearly?

If you want to reproduce, simply go to color picker on the primeng.org website (https://primeng.org/colorpicker) and select the Stackblitz for the Basic example

**Comment by rajabilal555** [comment]
Downgrading to `17.18.10` seems to be the solution for the time being...

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation and try the latest PrimeNG version(v19). If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,

**Comment by ThoSap** [comment]
Hi @mertsincan, this is still an issue.
It has been present since PrimeNG 17.18.11 and all subsequent releases.
Please fix this and **please** make sure to backport this fix to release PrimeNG 17.18.16 (**non-LTS**), many users can no longer use the ColorPicker since the 17.18.11 release 3 months ago without downloading the working images manually (see the working 17.18.10 links below) and overriding the global styles for class `.p-colorpicker-panel .p-colorpicker-color/hue`.

Related tickets:
https://github.com/primefaces/primeng/issues/16538
https://github.com/primefaces/primeng/issues/16586

The resources/images are working in PrimeNG **17.18.10**:
https://unpkg.com/browse/primeng@17.18.10/resources/images/
https://unpkg.com/browse/primeng@17.18.10/resources/images/color.png
https://unpkg.com/browse/primeng@17.18.10/resources/images/hue.png
https://unpkg.com/browse/primeng@17.18.10/resources/images/line.gif
https://unpkg.com/browse/primeng@17.18.10/resources/images/password-meter.png

They are no longer working/corrupted in PrimeNG (something must have changed in the release process):

- **17.18.11**
https://unpkg.com/browse/primeng@17.18.11/resources/images/
https://unpkg.com/browse/primeng@17.18.11/resources/images/color.png
https://unpkg.com/browse/primeng@17.18.11/resources/images/hue.png
https://unpkg.com/browse/primeng@17.18.11/resources/images/line.gif
https://unpkg.com/browse/primeng@17.18.11/resources/images/password-meter.png
- **17.18.12**
https://unpkg.com/browse/primeng@17.18.12/resources/images/
https://unpkg.com/browse/primeng@17.18.12/resources/images/color.png
https://unpkg.com/browse/primeng@17.18.12/resources/images/hue.png
https://unpkg.com/browse/primeng@17.18.12/resources/images/line.gif
https://unpkg.com/browse/primeng@17.18.12/resources/images/password-meter.png
- **17.18.13**
https://unpkg.com/browse/primeng@17.18.13/resources/images/
https://unpkg.com/browse/primeng@17.18.13/resources/images/color.png
https://unpkg.com/browse/primeng@17.18.13/resources/images/hue.png
https://unpkg.com/browse/primeng@17.18.13/resources/images/line.gif
https://unpkg.com/browse/primeng@17.18.13/resources/images/password-meter.png
- **17.18.14**
**This release is completely broken, compare https://unpkg.com/browse/primeng@17.18.14/ with https://unpkg.com/browse/primeng@17.18.15/**
https://unpkg.com/browse/primeng@17.18.14/resources/images/
https://unpkg.com/browse/primeng@17.18.14/resources/images/color.png
https://unpkg.com/browse/primeng@17.18.14/resources/images/hue.png
https://unpkg.com/browse/primeng@17.18.14/resources/images/line.gif
https://unpkg.com/browse/primeng@17.18.14/resources/images/password-meter.png
- **17.18.15**
https://unpkg.com/browse/primeng@17.18.15/resources/images/
https://unpkg.com/browse/primeng@17.18.15/resources/images/color.png
https://unpkg.com/browse/primeng@17.18.15/resources/images/hue.png
https://unpkg.com/browse/primeng@17.18.15/resources/images/line.gif
https://unpkg.com/browse/primeng@17.18.15/resources/images/password-meter.png

**Comment by cagataycivici** [comment]
@mertsincan @cetincakiroglu Guys, I'd appreciate if you can review tomorrow. There is a lot of demand from our users about this one. The brand new ColorPicker can wait, as we should make the current one right.

**Comment by mertsincan** [comment]
Could you please try;
```
.p-colorpicker-panel .p-colorpicker-color-background {
	width: 100%;
        height: 100%;
        background: linear-gradient(to top, #000 0%, rgba(0, 0, 0, 0) 100%), linear-gradient(to right, #fff 0%, rgba(255, 255, 255, 0) 100%) !important;
}


.p-colorpicker-panel .p-colorpicker-hue {
	background: linear-gradient(0deg, red 0, #ff0 17%, #0f0 33%, #0ff 50%, #00f 67%, #f0f 83%, red) !important;
}
```

**Comment by cagataycivici** [comment]
This will be automatically fixed in v20 since we're moving to shared primeuix, right now PrimeNG has its own fork. Still I think we should fix it for v19.1.x.
